main.go

```diff
@@ -103,6 +104,11 @@ func main() {
 		case "profile":
 			namespaceController := namespace.NewNamespaceController(ctx, k8sClientset, calicoClient)
 			go namespaceController.Run(config.ProfileWorkers, config.ReconcilerPeriod, stop)
+			if apiconfig.IsAlphaFeatureSet(os.Getenv("ALPHA_FEATURES"), apiconfig.AlphaFeatureSA) {
```

Comment:
that is what i originally thinking of doing too. But what made me do using env var setting is because:
- we already have `profile` and eventually `service accounts` will just fall under it.
- The setting with `ALPHA_FEATURES` seemed consistent with what we did earlier.

Implementation wise `serviceaccount` won't be part of the default setting for the kube-controller for now [here](https://github.com/projectcalico/kube-controllers/blob/1989bad7d23c22ae75ac0d752512c1f4a67bd98a/pkg/config/config.go#L32) and we'd replace `profile` with `namespace`. Correct?
But we'd still enable both for testing.

Reply:
Yeah, that's right. I think we'd still want to put it behind an alpha flag, but as far as how to enable the controller it should be a separate option in ENABLED_CONTROLLERS